client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -364,9 +366,15 @@ const formatMonitorData = ( site: Site ) => {
 		error: false,
 		settings: site.monitor_settings,
 	};
-	if ( ! site.monitor_active ) {
+	const monitorStatus = site.monitor_settings.monitor_active;
+	if ( ! monitorStatus ) {
 		monitor.status = 'disabled';
-	} else if ( ! site.monitor_site_status ) {
+	} else if (
+		! site.monitor_site_status &&
+		INITIAL_UNIX_EPOCH !== site.monitor_last_status_change
+		// This check is needed because monitor_site_status is false by default
+		// and we don't want to show the site down status when the site is first connected and the monitor is enabled
```

Comment:
Nitpick: I'd put this comment above the `INITIAL_UNIX_EPOCH` as IMO the comment is related to it.

Reply:
Thanks for the review, @vitozev! I have updated it - https://github.com/Automattic/wp-calypso/pull/71752/commits/776d567025ac312f808b28854f3b159bc41c2625.